### PR TITLE
add icon title

### PR DESCRIPTION
**Pull request by austinscreations**
will show information if you hover mouse over icon

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -473,23 +473,23 @@
 					if (data.network.mode === "wifi") 
 					{
 						// wifi is the network mode from adopt payload - add the wifi icon
-						$('#root_network').parent().append('<div id="icons"><i class="mdi mdi-wifi" id="wifi-icon" aria-hidden="true" style="color:#2C448E;font-size: 30px;"></i></div>');
+						$('#root_network').parent().append('<div id="icons"><i class="mdi mdi-wifi" id="wifi-icon" aria-hidden="true" title="Device Using Wifi" style="color:#2C448E;font-size: 30px;"></i></div>');
```
Comment (sumnerboy12) [inline index.html]: I think I would prefer to see two local variables defined, one for the icon name and one for the title. Then have a single line to add the `<div>` etc, substituting those variables.

For one it means all that complex HTML code is only defined once. 

Then I would just do `var title = data.network.mode`. So the tooltip is the raw network mode from the adoption payload.

**Review by sumnerboy12 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (austinscreations) [reply index.html]: I'm not sure that's possible

**Review by sumnerboy12 — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -468,31 +468,26 @@
 				// remove prior icons - otherwise it'll duplicate
 				$('#icons').remove();
 				// if information is unavailble this prevents issues with jquery
-				if (data.hasOwnProperty("network") && data.network.hasOwnProperty("mode"))
+				if (data.hasOwnProperty("network"))
```
Comment (sumnerboy12) [inline index.html:471]: This check isn't needed - as we have already set `firmwareData.network` in all cases above.

File: index.html [hunk 3]
```diff
@@ -468,31 +468,26 @@
...
-					{
-						// wifi is the network mode from adopt payload - add the wifi icon
-						$('#root_network').parent().append('<div id="icons"><i class="mdi mdi-wifi" id="wifi-icon" aria-hidden="true" style="color:#2C448E;font-size: 30px;"></i></div>');
-					}
-					else if (data.network.mode === "ethernet") 
-					{
-						// ethernet is the network mode from adopt payload - add the ethernet icon
-						$('#root_network').parent().append('<div id="icons"><i class="mdi mdi-ethernet" id="ethernet-icon" aria-hidden="true" style="color:#2C448E;font-size: 30px;"></i></div>');
+					// dynamically set the title of the icon with adopt network mode information
+					var titleFirmware = firmwareData.network;
```
Comment (sumnerboy12) [inline index.html]: Are these variables not scoped to this function?

**Review by sumnerboy12 — CHANGES_REQUESTED** [review]

**Pull request merged**